app/controllers/follower_accounts_controller.rb

```diff
@@ -4,11 +4,9 @@ class FollowerAccountsController < ApplicationController
   include AccountControllerConcern
 
   def index
-    @follows = Follow.where(target_account: @account).recent.page(params[:page]).per(FOLLOW_PER_PAGE).preload(:account)
-
     respond_to do |format|
       format.html do
-        @relationships = AccountRelationshipsPresenter.new(@follows.map(&:account_id), current_user.account_id) if user_signed_in?
+        @relationships = AccountRelationshipsPresenter.new(follows.map(&:account_id), current_user.account_id) if user_signed_in?
```

Comment:
If `user_signed_in?` is false, `follows` is never called, so `@follows` is not assigned, but it is required for the HTML views.

```
ActionView::Template::Error:
       undefined method `map' for nil:NilClass
```

Reply:
Far easier solution is just to call follows once above this line

Reply:
Sorry about that. Fixed it.